lang/eo/strings.po

```diff
@@ -1340,75 +1350,75 @@ msgstr "Malnova&nbsp;Domo"
 
 #: script/world.js:124
 msgid "A&nbsp;Damp&nbsp;Cave"
-msgstr ""
+msgstr "Malseka Kaverno"
```

Comment:
Those strings need the "&nbsp;" as a separator to keep it in a single line of text. I've corrected it in another commit.